docs/kubernetes.md

```diff
@@ -0,0 +1,101 @@
+# Running on Kubernetes
+
+Daft can run on Kubernetes using the Daft quickstart Helm chart. This allows you to test drive both simple single-node jobs and distributed Ray-based workloads on your Kubernetes cluster without any additional dependencies.
+
+!!! warning "Not for Production"
```

Comment:
Let's just remove this -- I'd love for folks to not have to think about KubeRay when using daft on k8s